app/scripts/modules/kubernetes/src/v2/serverGroupManager/details/details.html

```diff
@@ -97,6 +97,8 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
       </ul>
     </collapsible-section>
 
+    <kubernetes-annotation-custom-sections manifest="ctrl.serverGroupManager.manifest" resource="ctrl.serverGroupManager"></kubernetes-annotation-custom-sections>
```

Comment:
Can you add this to the instance details as well?

Reply:
What `instance` are you referring to here?  I have added it to app/scripts/modules/kubernetes/src/v2/instance/details/details.html as part of this PR.  is there somewhere else to include it?

Reply:
ping